soc/riscv/esp32c3/Kconfig.soc

```diff
@@ -30,4 +30,16 @@ config FLASH_BASE_ADDRESS
 	hex
 	default $(dt_node_reg_addr_hex,/soc/flash-controller@60002000/flash@0)
 
+config ESP32_PHY_MAX_WIFI_TX_POWER
+	int "Max WiFi TX power (dBm)"
```

Comment:
Although naming suggests "WiFi" configuration, this needs to be placed in `soc` level because it is related to PHY. Internal calls in `hal_espressif` requires this value to initialize its subsystems.